README.rst

```diff
@@ -272,3 +272,12 @@ As in the case of abbreviations, you can hear the audio for a given sentence and
 .. _tab separated file: https://en.wikipedia.org/wiki/Tab-separated_values
 .. _common.py: https://github.com/mozilla/CorporaCreator/blob/master/src/corporacreator/preprocessors/common.py
 .. _en.py: https://github.com/mozilla/CorporaCreator/blob/master/src/corporacreator/preprocessors/en.py
+
+
+Development
```

Comment:
Dependencies should be added to setup.cfg. See the documentation [here](https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files). So this should be removed from the readme.

The reademe already contains info on [installation](https://github.com/mozilla/CorporaCreator#installation).